.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,33 @@
+# Go to https://github.com/actions/stale for the full documentation of these options.
+
+name: 'Stale bot'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3.0.18
+        with:
+          # Rate limit per run, defaults to 30.
+          operations-per-run: 30
+
+          # Only issues with both `reproduction:needed` and `type:bug` will be touched by the stale bot.
+          only-issue-labels: 'reproduction:needed,type:bug'
+
+          # A issue that has any assignee is getting worked on, so is exempt from stale bot.
+          exempt-all-issue-assignees: true
+
+          # Remove stale label from issue/PR on updates or comments, defaults to true.
+          remove-stale-when-updated: true
+
+          # Wait 14 days until making issue stale and posting a message.
+          days-before-issue-stale: 14
+          stale-issue-message: 'This issue will be closed after 7 days if a reproduction is not provided.'
```

Comment:
Let's make this more descriptive and polite. e.g. When a bug has been marked as needing a reproduction, it means nobody can work on it until one is provided. In cases where no reproduction is possible, or the issue creator does not have the time to reproduce, we unfortunately need to close such issues as they are non-actionable and serve no benefit by remaining open.

Reply:
```suggestion
          stale-issue-message: 'When a bug has been marked as needing a reproduction, it means nobody can work on it until one is provided. In cases where no reproduction is possible, or the issue creator does not have the time to reproduce, we unfortunately need to close such issues as they are non-actionable and serve no benefit by remaining open.'
```

Reply:
We forgot to add the 7 day info back :)